chrome-launcher/chrome-launcher.ts

```diff
@@ -46,22 +48,35 @@ export interface ModuleOverrides {
   spawn?: typeof childProcess.spawn;
 }
 
+const sigintListener = async () => {
+  for (const instance of instances) {
+    await instance.kill();
+  }
+  process.exit(_SIGINT_EXIT_CODE);
+};
+
 export async function launch(opts: Options = {}): Promise<LaunchedChrome> {
   opts.handleSIGINT = defaults(opts.handleSIGINT, true);
 
   const instance = new Launcher(opts);
 
   // Kill spawned Chrome process in case of ctrl-C.
-  if (opts.handleSIGINT) {
-    process.on(_SIGINT, async () => {
-      await instance.kill();
-      process.exit(_SIGINT_EXIT_CODE);
-    });
+  if (opts.handleSIGINT && instances.size === 0) {
+    process.on(_SIGINT, sigintListener);
```

Comment:
@paulirish @JoelEinbinder this gonna yield "max listeners" error in case of 11 simultaneously launched browsers.

Reply:
instances is a global variable, so it only adds one listener per project.

Reply:
ah i see. So the "handleSIGINT" option is not per-browser but per-process.